packages/svelte/tests/compiler-errors/samples/runes-non-bindable-prop-mutated/main.svelte

```diff
@@ -0,0 +1,4 @@
+<script>
+	let { x } = $props();
+	x.y = 1;
```

Comment:
yay or nay for throwing a compiler error in this case? The argument against it would be setters on a class, but I'm wondering how big of a deal it really is to make the prop bindable in that case, since this will in the vast majority of cases guide people towards the right outcome.